commands/file_operation_unix.go

```diff
@@ -0,0 +1,52 @@
+// +build !windows
```

Comment:
What does this mean? Where is license also?

Reply:
Good catch. I assumed there was a license check. Created an issue here https://github.com/opensearch-project/opensearch-cli/issues/37

Reply:
This line means, this file will only be included during build if it is not windows. Hence, in other file i added just for windows.

Reply:
Oh thats pretty cool.